test/testlib/nuodb_database_utilities.go

```diff
@@ -46,6 +47,11 @@ func GetExtractedOptions(options *helm.Options) (opt ExtractedOptions) {
 		opt.DbName = "demo"
 	}
 
+	opt.ClusterName = options.SetValues["cloud.clusterName"]
```

Comment:
can you add this logic to `minikube_test_test.go`?

Reply:
Sure! :)